migrations/20201009045236-viewing.ts

```diff
@@ -0,0 +1,36 @@
+import Base from "db-migrate-base";
+
+export async function up(
```

Comment:
can you add a comment above the up function describing what this migration does when you run it? (eg smth like how it creates the viewing permissions table along with where the foreign keys are from